jnigen/tool/pr_checks.dart

```diff
@@ -16,26 +16,46 @@
 import 'dart:async';
```

Comment:
Should we run this script on the CI to check that we don't break the script accidentally?

Or does this try do things not supported on github bots?